packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -876,6 +883,19 @@ export class GarbageCollector implements IGarbageCollector {
             this.runtime.deleteUnusedRoutes(gcResult.deletedNodeIds);
         }
 
+        if (this.tombstoneMode) {
+            const tombstoneableRoutes: string[] = [];
+            for (const [key, value] of this.unreferencedNodesState.entries()) {
+                if (
```

Comment:
I might put a comment here calling out that we're only doing this for DataStores for now, not attachment blobs

Reply:
Sure